chore(): update acurast intro and about (#107)
moondance-labs/tanssi-docs: builders/tooling/oracles/acurast.md

## Introduction {: #introduction }

[Acurast](https://acurast.com/){target=\_blank} gives developers complete permisionless access to compute that is trustless, affordable and confidential for deploying their applications.

> One of the Acurast's use cases is to enable developers to deploy their own push/pull oracles, interacting with off-chain APIs to bring e.g., price feeds on-chain. Pricing data is confidentially processed through Acurast Processors, pushing data to smart contracts of EVM-compatible chains like Tanssi EVM Appchains to e.g., a [standard Chainlink Aggregator Interface](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol){target=\_blank} smart contract.

Reviewer: ```suggestion
One of Acurast's use cases is to enable developers to deploy their own push/pull oracles, interacting with off-chain APIs to bring price feeds on-chain. Pricing data is confidentially processed through Acurast Processors, pushing data to smart contracts of EVM-compatible chains like Tanssi EVM Appchains via a [standard Chainlink Aggregator Interface](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol){target=\_blank}.
```

This tutorial will walk through a demo of [interacting with price feeds](#fetch-price-data) enabled by Acurast on the [demo Tanssi EVM-compatible Appchain](/builders/tanssi-network/networks/dancebox/demo-evm-containerchain/){target=\_blank}. You can also deploy your own price feeds to your Tanssi EVM-compatible Appchain. Please be advised that the steps shown in this tutorial are for demonstration purposes only - it's highly recommended that you contact the [Acurast](https://acurast.com/){target=\_blank} team directly as they can assist you with launching price feeds on your Appchain to ensure the integrity of the deployment process.

## What is Acurast? {: #what-is-acurast }

Acurast is a decentralized, serverless cloud where everyone can become part of the cloud with their new, used, or even mobile phones with a smashed screen by providing compute power to the cloud and earning rewards. These so-called Processors are scattered across the globe, creating a distributed network of compute across the globe.

> Giving developers complete permisionless access to compute that is trustless, affordable and confidential for deploying their applications like price feed oracles.

Reviewer: ```suggestion
Acurast gives developers complete permissionless access to compute that is trustless, affordable and confidential for deploying their applications like price feed oracles.
```

> Processors and developers seamlessly interact through the [Acurast Console](https://console.acurast.com/){target=\_blank} with the Acurast protocol.

Reviewer: ```suggestion
Processors and developers can seamlessly interact through the [Acurast Console](https://console.acurast.com/){target=\_blank}.
```

## Introduction {: #introduction }

[Acurast](https://acurast.com/){target=\_blank} gives developers complete permisionless access to compute that is trustless, affordable and confidential for deploying their applications.

> One of the Acurast's use cases is to enable developers to deploy their own push/pull oracles, interacting with off-chain APIs to bring e.g., price feeds on-chain. Pricing data is confidentially processed through Acurast Processors, pushing data to smart contracts of EVM-compatible chains like Tanssi EVM Appchains to e.g., a [standard Chainlink Aggregator Interface](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol){target=\_blank} smart contract.

Reviewer: This looks great

This tutorial will walk through a demo of [interacting with price feeds](#fetch-price-data) enabled by Acurast on the [demo Tanssi EVM-compatible Appchain](/builders/tanssi-network/networks/dancebox/demo-evm-containerchain/){target=\_blank}. You can also deploy your own price feeds to your Tanssi EVM-compatible Appchain. Please be advised that the steps shown in this tutorial are for demonstration purposes only - it's highly recommended that you contact the [Acurast](https://acurast.com/){target=\_blank} team directly as they can assist you with launching price feeds on your Appchain to ensure the integrity of the deployment process.

## What is Acurast? {: #what-is-acurast }

> Acurast, the decentralized serverless cloud.

Reviewer: ```suggestion
Acurast is a decentralized, serverless cloud.
```

> The cloud is dominated by three main players, this centralization comes at a price: high cost of compute, exclusion of less developed parts of the world, and the hidden cost, loss of data ownership. Acurast is solving all three problems by bringing the convenience of a cloud - whilst giving users full confidentiality over their data - at a fraction of the cost.

Reviewer: Please revise this to make it more informational and less like a pitch. We try to maintain a neutral tone in our docs rather than taking a side.

> To truly disrupt, when competing against giants, one cannot play by the same rules, neither use same tools to win.

Reviewer: This comes across as copy rather than informational text for developers. Please revise the wording or can remove.
